pkg/utils/remediationutils.go

```diff
@@ -20,6 +21,16 @@ func IsMachineConfig(obj *unstructured.Unstructured) bool {
 	return "MachineConfig" == objgvk.Kind && mcfgapi.GroupName == objgvk.Group
 }
 
+func IsKubeletConfig(obj *unstructured.Unstructured) bool {
```

Comment:
to avoid code duplication, I think we could have a common function for this. e.g.

```
func IsKind(obj *unstructured.Unstructured, kind string) bool {
	if obj == nil {
		return false
	}
	// FIXME(jaosorior): Find a more dynamic way to get
	// the MachineConfig's GVK
	objgvk := obj.GroupVersionKind()
	return kind == objgvk.Kind && mcfgapi.GroupName == objgvk.Group
}
```

So then you could wrap `IsKubeletConfig` as follows:
```
func IsKubeletConfig(obj *unstructured.Unstructured) bool {
  return IsKind(obj, "KubeletConfig")
}

Reply:
That's a good idea!